.circleci/scripts/package-validation/tf/main.tf

```diff
@@ -0,0 +1,108 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 2.70"
+    }
+  }
+}
+
+variable "identifier" {
+  type = string
+}
+
+variable "package_path" {
+  type = string
+}
+
+provider "aws" {
+  region = "us-east-1"
+}
+
+data "aws_ami" "centos" {
+  most_recent = true
+
+  # This information is sourced from https://wiki.centos.org/Cloud/AWS
+  # and should pull the latest AWS-provided CentOS 7 image.
+  filter {
+    name   = "name"
+    values = ["CentOS Stream 9 x86_64*"]
```

Comment:
Do we want Stream or 7? I'm not sure if it matters one way or the other, just noticed the comment doesn't match the code.

Reply:
Thanks for catching this! I originally used the CentOS 7, but it turns out CPE has disabled the yum repositories. It ended up being simpler to use CentOS 9 Stream instead. I'll push a commit that fixes the comment.